frontend/src/__mocks__/mockDashboardConfig.ts

```diff
@@ -53,6 +54,7 @@ export const mockDashboardConfig = ({
   disableTracking = false,
   disableBYONImageStream = false,
   disableISVBadges = false,
+  genAiStudio = true,
```

Comment:
Shouldn't this be the inverse now and start off disabled (`false`)?
What about `modelAsService` ?